tidb-in-kubernetes/maintain/restart.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Restart a TiDB Cluster in Kubernetes
+summary: Learn how to restart a TiDB cluster in the Kubernetes cluster.
+category: how-to
+---
+
+# Restart a TiDB Cluster in Kubernetes
+
+This document describes how to forcibly restart a TiDB cluster in the Kubernetes cluster, including restarting a Pod, restarting all Pods of a component, and restarting all Pods of the TiDB cluster.
+
+> **Note:**
+>
+> TiDB Operator v1.0.x only supports restarting the Pod in a forcible way.
+>
+> - If the PD Pod being restarted is the PD leader, its leadership does not automatically shift, which can cause the PD service to be temporarily unavailable.
+> - If the TiKV Pod being restarted contains a Region leader of the TiKV cluster, its leadership does not automatically shift, which can cause the requests of accessing the corresponding data to fail.
```

Comment:
From our current documentation, usually, we say leader `transfer` instead of `shift`.